mitiq/interface/utils.py

```diff
@@ -138,6 +138,14 @@ def _count_gate_arities_qibo(circuit: Any) -> dict[str, int]:
     return counts
 
 
+# TODO: Temporary solution
```

Comment:
Can you leave a more descriptive note as to what needs to be improved here if it's temporary? Looks okay to me tbh.